appveyor.yml

```diff
@@ -20,10 +20,19 @@ init:
   - conda config --set always_yes yes
   # We need to do this first as other commands may not work with older versions of conda.
   - conda update conda
-  - conda install numpy scipy nose hdf5 --quiet
+  - conda install numpy scipy nose hdf5 sphinx mock --quiet
 
 install:
+  - echo on
+  #We need to run these two lines to have nmake available for doctests
+  - cd "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC"
+  - vcvarsall amd64
+  #Return back to continue install of OpenMDAO
+  - cd C:\projects\openmdao*
```

Comment:
@swryan: wildcarding worked, looks like this is good to go.  terminal output confirms build and test of docs.